source/js/plugins/_filters.js

```diff
@@ -66,7 +132,7 @@ function _apply (data, pixelFunc, adjust)
  **/
 export function grayscale(data)
 {
-    return _apply(data, _grayscale);
+    return addFilterToQueue(_grayscale, data);
```

Comment:
You should keep the argument order consistent.

Reply:
Sure